bioconda_utils/utils.py

```diff
@@ -639,3 +639,26 @@ def get_list(key):
 
     default_config.update(config)
     return default_config
+
+
+def modified_recipes(git_range, recipe_folder):
```

Comment:
@johanneskoester some changes here that might be useful, specifically making `--relative` be the provided recipe dir, and using `shell=True` which allows the wildcards to be expanded. I think the existing code in master looks for the literal filename "recipes/*/meta.yaml" without expanding the glob.